wordpress-mobile/WordPress-iOS
# Analytics: Adds a counter to help investigate an old stats issue.

Refs #6819 

This PR adds a helper class to investigate excessive load more calls. We'll bump the stat once so we know its working.  Then we'll bump again when the counter reaches 100, 1000, and 10000.  Each new instance of a post list gets a fresh copy of the helper class. 

To test: 
- Run the tests and confirm they pass. 
- Open the app to a post list with enough posts to trigger a load more action. 
- Scroll down to sync more posts. 
- Wait a few minutes, then check for a single instance of the new stat to appear in tracks. The count should be `1`.
<img width="660" alt="screen shot 2019-02-21 at 5 16 26 pm" src="https://user-images.githubusercontent.com/1435271/53208466-786dc000-35fc-11e9-91f5-d96749ecc74c.png">


Heya, @stevebaranski.  This is the WPiOS counterpart to the earlier shared pod patch. Still game for a review?
